numba/core/analysis.py

```diff
@@ -397,6 +397,7 @@ def resolve_input_arg_const(input_arg_idx):
     # at least one an arg of the condition is a const
     # if the condition is met it will replace the branch with a jump
     branch_info = find_branches(func_ir)
+    nullified = namedtuple('nullified', 'condition, taken_br, rewrite_stmt')
```

Comment:
I'd prefer to move the `namedtuple()` out to global as it's quite an expensive operation.

Reply:
Good point, done.